api/src/main/java/org/openmrs/module/commonreports/reports/MSPPVisitsReportManager.java

```diff
@@ -138,19 +138,21 @@ private String applyMetadataReplacements(String rawSql) {
 	private Map<String, String> getMetadataReplacements() {
 		Map<String, String> map = new HashMap<String, String>();
 		String prenatalVisitTypeUuid = inizService.getValueFromKey("report.MSPP.visits.prenatal.visitType.uuid");
+		String postnatalVisitTypeUuid = inizService.getValueFromKey("report.MSPP.visits.postnatal.visitType.uuid");
```

Comment:
Have you specified this property in the distro?

Reply:
Yes. https://github.com/mekomsolutions/bahmni-distro-haiti/commit/d2a0d8ff5c5a09c51ed07ed79ed8e3ebdf259a5f#diff-0fd05f298f55b6887d1fe0b9f32ea24953c0ad2bac16fde947f10f06f908a775R3-R4